Apps/Examples/Examples/All Examples/DebugMapExample.swift

```diff
@@ -0,0 +1,183 @@
+import UIKit
+import MapboxMaps
+
+private protocol DebugOptionSettingsDelegate: AnyObject {
+    func debugOptionSettingsDidChange(_ controller: SettingsViewController)
+}
+
+private struct MapDebugOptionSetting {
+    let debugOption: MapDebugOptions
+    let displayTitle: String
+}
+
+final class DebugMapExample: UIViewController, ExampleProtocol, DebugOptionSettingsDelegate {
+
+    private var mapView: MapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MapView(frame: view.bounds)
+        view.addSubview(mapView)
+
+        mapView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            mapView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            mapView.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor),
+            mapView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            mapView.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor),
+        ])
+
+        let debugOptionsBarItem = UIBarButtonItem(
+            barButtonSystemItem: .edit,
+            target: self,
+            action: #selector(openDebugOptionsMenu(_:)))
+        navigationItem.rightBarButtonItems?.insert(debugOptionsBarItem, at: 0)
```

Comment:
One more thing - don't forget to call `finish()` the end of `viewDidLoad()`. Unit tests expect `finish()` to be called sometime after the example is set up, and changes in #1215 will fail the test if this expectation is not met.